src/components/Meter.jsx

```diff
@@ -11,7 +11,18 @@ function Meter(props) {
     // Different heights for level = 1, 2, 3, 4
     const lowLevelHeights = [50, 65, 80, 93];
 
-    const title = type == "sunlight" ? "Sunlight Exposure" : "Soil Moisture";
+    let title;
+
+    switch (type) {
+        case "sunlight":
+            title = "Sunlight Exposure";
+            break;
+        case "moisture":
+            title = "Soil Moisture";
+            break;
+        default:
+            title = "Temperature";
```

Comment:
I think you should be specific about what should render a temperature meter and throw an error if you don't recognise the meter type.

This will give you a more meaningful response to garbage data than silently handling it.

```suggestion
        case "temperature":  // or whatever the correct string is
            title = "Temperature";
            break;
        default:
            throw new Error("Unknown meter type"); // or whatever error message you prefer
```

If you want to know more about this suggestion, look into switch statement exhaustiveness checking.